x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -325,9 +332,30 @@ function waitForKibanaLogin(kbUrl: string, credentials: Credentials): Promise<vo
   return pRetry(fetchLoginStatusAttempt, retryOptions);
 }
 
+const getOverridedProductTypes = (
+  projectConfigurationParameters: ProjectConfigurationParameters
```

Comment:
NIT: destructing the parameters will make the implementation more readable
```suggestion
  { tier, endpointAddon, cloudAddon }: ProjectConfigurationParameters
```